docs/content/colors.mdx

```diff
@@ -24,6 +24,12 @@ Preview color variables in any of the available themes:
 
 <ColorThemePicker />
 
+### Color vision deficiency themes
+Primer primitives include `Protanopia & Deuteranopia` and `Tritanopia` themes for dark and light mode.  
+In these themes, green, red and orange are changed to different colors. This allows us to provide the right color contrast in the UI for people with color vision deficiency (cvd). For example, when comparing branches in pull requests.
+
+When changing themes, we adjust the scales instead of changing which scale a functional tokens references. This results in a mismatch of color scale names and the value they hold. For example, the green scale uses blue in cvd modes. This is not a bug but is desired for easier maintenance. Color scales should not be used directly, so this mismatch is an internal implementation detail not visible to out users.
```

Comment:
Nitpick suggestion for clarity

```suggestion
When changing themes, we adjust the scales instead of the functional tokens references. This results in a mismatch of color scale names and the value they hold. For example, the green scale uses blue in CVD modes. This is not a bug but is desired for easier maintenance. Color scales should not be used directly, so this mismatch remains invisible to our users.
```

Reply:
Hey @josepmartins I am not 100% sure this makes it more clear. 

How about:

`instead of changing the functional references`

and

`so this mismatch is not visible to our users`

Reply:
👍 like it